sdk/textanalytics/tests.yml

```diff
@@ -4,5 +4,7 @@ extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: textanalytics
-    Clouds: 'Preview'
+    CloudConfig:
```

Comment:
@benbp 
I imagine Public cloud will still run, right? or by doing this it is overriding the whole CloudConfig object?

Reply:
trying another approach